### ISSUE TITLE
object mysteriously losing velocity

### ISSUE THREAD

**Issue by spiderworm** [shakiba/planck.js#82]
I'm having a mysterious problem, I need some help figuring this out. Using Planck 0.3.5 from NPM

1. I have a world with 0 gravity
2. I have created a dynamic body, I am giving it an initial linear velocity of Vec2(0, 10000)
3. I check body.getLinearVelocity(), which validates that it's still Vec2(0, 10000)
3. I step the world: world.step(.001)
4. I then check body.getLinearVelocity() again, but now velocity is Vec2(0, 2000)

What could be causing this?

I have already validated a number of things:
- body.getLinearDamping() returns 0
- body.getAngularDamping() returns 0
- body.getMass() returns 45364597.917836614
- world.getBodyCount() returns 1

???

**Comment by shakiba** [comment]
Can you please share your code like [this](https://jsbin.com/nujuxik/edit?html,output)?

**Comment by spiderworm** [comment]
@shakiba https://jsbin.com/rerilikepi/edit?html,output

**Comment by spiderworm** [comment]
Well now, this is interesting. https://jsbin.com/rerilikepi/edit?html,output

If I modify the STEP_SIZE constant, the velocity changes differently. You should try it, with a bunch of different values, like .1, .00001, etc.

What units is velocity supposed to be in? Right now it looks like you're expressing it relative to the size of the last world step.

**Comment by shakiba** [comment]
Yes, that's actually how it works, see [here](https://stackoverflow.com/questions/14774202/is-there-an-upper-limit-on-velocity-when-using-box2d).

**Comment by shakiba** [comment]
Also Box2D [FAQ](https://github.com/erincatto/Box2D/wiki/FAQ) is very helpful.

**Comment by spiderworm** [comment]
Using this information, I was able to discover that by scaling down the size, positions, and velocity of anything I passed into planck by a factor of .008 (so that my physics bodies in planck would be of size 0.1 to 2.0) everything started working better.

It would be nice if planck complained or warned about physics bodies that are much larger than that. 

It would also be nice if planck provided a "scale" setting, so that I could just set scale once on a planck World and it does all the necessary additions/multiplications automagically for me.
